components/FAQ/FAQAccordion.tsx

```diff
@@ -0,0 +1,92 @@
+import LinkButton from '@/LinkButton'
+import React from 'react'
+
+
+export default function FAQDropDown() {
+  return (
+    <div className="grid grid-cols-2 gap-10 p-10">
+
+      <div style={{ display: 'flex', flexDirection: 'column', justifyContent: 'space-between' }}>
+        <h2>Често поставувани
```

Comment:
Implementation doesn't match the design. Text should be bold and larger in size.